### PR TITLE
updating the readme with openscap details

### PR DESCRIPTION
**Pull request by enoodle**
I forgot to do this before..

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +44,7 @@ To build image-inspector using godep:
 
 # Running as a container
 
-    $ docker run -ti --rm --privileged \
+    $ docker run -ti --rm --privileged -p 5000:5000 \
       -v /var/run/docker.sock:/var/run/docker.sock \
-      openshift/image-inspector --image=fedora:20 --path=/tmp/image-content
+      openshift/image-inspector --image=fedora:20 \
+      --path=/tmp/image-content --serve 0.0.0.:5000
```
Comment (simon3z) [inline README.md]: @enoodle typo `0.0.0.`?

File: README.md [hunk 2]
```diff
@@ -29,6 +44,7 @@ To build image-inspector using godep:
 
 # Running as a container
 
-    $ docker run -ti --rm --privileged \
+    $ docker run -ti --rm --privileged -p 5000:5000 \
```
Comment (simon3z) [inline README.md]: @enoodle I have the feeling that when this is consumed we use `8080` more often. Anyway it's a nit/taste.

**Comment by simon3z** [comment]
@pweil- can you review? (Also the text if needs to be clearer, etc.)

File: README.md (hunk 2 above)
Comment (pweil-) [inline README.md]: agree, the first run example in the readme (line 6) uses 8080 so I'd probably keep it consistent.

**Comment by pweil-** [comment]
wording looks good to me

File: README.md [hunk 3]
```diff
@@ -21,6 +21,21 @@ Image Inspector can extract docker images to a target directory and
...
+the OpenSCAP section.
+
+    $ sudo ./image-inspector --image=fedora:22 --path=/tmp/image-content --scan-type=openscap
+			--serve 0.0.0.0:8080 --chroot
+    2016/05/25 16:12:04 Image fedora:22 is available, skipping image pull
+    2016/05/25 16:12:04 Extracting image fedora:22 to /tmp/image-content
+    2016/05/25 16:12:14 OpenSCAP scanning /tmp/image-content. 
+						Placing results in /var/tmp/image-inspector-scan-results-845509636
+    2016/05/25 16:12:20 Serving image content /tmp/image-content on 
+						webdav://0.0.0.0:8080/api/v1/content/
```
Comment (simon3z) [inline README.md]: @enoodle can you remove the long white-spaces indentation for "Placing results in..." and "webdav://0.0.0.0...". (Or maybe there's a reason for it?) Thanks.

**Comment by simon3z** [comment]
@enoodle did you revert this back to use port `5000`? Why?

**Comment by enoodle** [comment]
I am sorry, I fixed a comment from a different computer and forgot to pull before. I will fix that again.

**Pull request merged**